.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: php
+php:
+  - 7.0
+  - 7.1
+before_install:
+    - composer self-update
+install:
+    - composer --prefer-source -n install
```

Comment:
you should remove `--prefer-source`. this makes the build slower, and github removed the API rate limit on the download endpoint 1 year ago